README.md

```diff
@@ -61,7 +61,7 @@ AzCopy supports two types of authentication. See the table below to see which ty
 | ------------- | ------------- |
 | Blob storage  | SAS tokens OR Azure Active Directory Authentication  |
 | File storage  | SAS tokens  |
-| ADLS Gen2 | Azure Active Directory Authentication |
+| ADLS Gen2 | SAS tokens OR Azure Active Directory Authentication |
```

Comment:
Can you please also make the corresponding change (adding SAS token as on option) to line 25 of helpMessages.go